src/views/language/LanguageCard.tsx

```diff
@@ -48,8 +48,8 @@ const LanguageCard: React.FC<Props> = ({ lang, includeRelations }) => {
         </div>
       )}
       <div>
-        <h4>Vitality</h4>
-        <LanguageVitalityMeter value={vitalityEth2013} />
+        <h4>Vitality Metascore</h4>
```

Comment:
You can just call it vitality.

```suggestion
        <h4>Vitality</h4>
```